src/envoy/http/jwt_auth/pubkey_cache.h

```diff
@@ -27,6 +27,10 @@ namespace JwtAuth {
 namespace {
 // Default cache expiration time in 5 minutes.
 const int kPubkeyCacheExpirationSec = 600;
+
+const std::string kHTTPSchemePrefix("http://");
```

Comment:
add comment per style guide for each global variable

Reply:
Done.